<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BioDialog</class>
 <widget class="QDialog" name="BioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Candidate Bio</string>
  </property>
  <property name="styleSheet">
   <string notr="true">     background-color:white;
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	border-image: url(:/head/Resources/headshot_dc-celebrity-portrait-photographers.jpg)
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLabel" name="fromLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="partyLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Party</string>
   </property>
  </widget>
  <widget class="QLabel" name="bioLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Candidate bio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:red;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}
</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="jobLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Job</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
